### PR TITLE
made home screen responsive for small screen sizes

### LINKED ISSUE #204: Make responsive for small screen sizes

### PR DESCRIPTION
**Pull request by sakshi2707**
closes #204

### CODE REVIEW

**Comment by sakshi2707** [comment]
@anushbhatia  please review and merge this PR

**Comment by anushbhatia** [comment]
Could you attach a screenshot of smaller devices.

**Comment by sakshi2707** [comment]
@anushbhatia done

**Comment by anushbhatia** [comment]
resolve the conflict screenshot is fine to me.

**Comment by sakshi2707** [comment]
there is no conflict showing in command line. Can you please help what to do?
Screenshot (1113)

**Comment by Saransh-cpp** [comment]
> there is no conflict showing in command line. Can you please help what to do?
> Screenshot (1113)

Pull the latest code from the original repo ([this](https://github.com/smaranjitghose/DocLense) one), not the forked repo, and then resolve conflicts.

**Comment by sakshi2707** [comment]
@Saransh-cpp Thankyou So much! 
@anushbhatia please add gssoc and level labels and merge it. Thankyou.

File: lib/home.dart [hunk 1]
```diff
@@ -1,8 +1,14 @@
 import 'dart:async';
 import 'dart:io';
 
+
+import 'package:doclense/Services/SearchService.dart';
```
Comment (Saransh-cpp) [inline lib/home.dart]: Replace
```dart
import 'package:doclense/Services/SearchService.dart';
```
with 
```dart
import 'package:doclense/Services/search_service.dart';
```

File: lib/home.dart [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
+import 'package:doclense/StarredDocuments.dart';
```
Comment (Saransh-cpp) [inline lib/home.dart]: Replace
```dart
import 'package:doclense/Services/StarredDocuments.dart';
```
with
```dart
import 'package:doclense/Services/starred_documents.dart';
```

**Review by Saransh-cpp — CHANGES_REQUESTED** [review]
After the following changes the lint tests will pass.

**Comment by sakshi2707** [comment]
Thankyou @anushbhatia . please add gssoc label also in the pr

**Pull request merged**